packages/dashboard/src/app/api/useTemplateApi.js

```diff
@@ -172,8 +172,21 @@ const useTemplateApi = (dataAdapter, config) => {
       payload: true,
     });
 
+    const templatesByTag = {};
+
     const reshapedTemplates = (await getAllTemplates({ cdnURL }))
-      .map((template) => reshapeTemplateObject(template, cdnURL))
+      .map((template) => {
+        const reshapedTemplate = reshapeTemplateObject(template, cdnURL);
+
+        reshapedTemplate.tags.forEach((tag) => {
+          if (templatesByTag[tag]) {
+            templatesByTag[tag].push(reshapedTemplate);
+          } else {
+            templatesByTag[tag] = [reshapedTemplate];
+          }
+        });
+        return reshapedTemplate;
+      })
```

Comment:
Figured it was easier to build an object that's organized by tag while we're already looping over all of the templates (on fetch). `templatesByTag` gets set in state for later use.